package.json

```diff
@@ -29,7 +29,8 @@
     "nyc": "10.1.2"
   },
   "dependencies": {
-    "apollo-client": "0.8.5",
+    "apollo-client": "^0.8.7",
```

Comment:
If you have to change the dependencies, please use `~` and while you're at it, can you add the `~` to the others as well ?